File: VB/WindowsFormsApplication88/CustomPopupColorBuilder.vb
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Popup
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication88
    Public Class CustomPopupColorBuilder
        Inherits PopupColorBuilderEx

        Public Sub New(ByVal owner As IPopupColorEdit)
            MyBase.New(owner)

        End Sub
        Protected Overrides Function CreateCustomTabInnerControl() As InnerColorPickControl
            Return New CustomInnerColorPickControl()
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication88/CustomPopupColorBuilder.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Popup;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication88 {
    public class CustomPopupColorBuilder : PopupColorBuilderEx {
        public CustomPopupColorBuilder(IPopupColorEdit owner) : base(owner) {

        }
        protected override InnerColorPickControl CreateCustomTabInnerControl() {
            return new CustomInnerColorPickControl();
        }
    }
}
